### PR TITLE
Add users to project

### PR DESCRIPTION
**Pull request by djino97**
Added new endpoint -> addUsersToProject
Task -- http://outsourcing.nat.tepkom.ru:20080/redmine/issues/66

### CODE REVIEW

File: src/ProjectService.Validation/AddUsersToProjectValidator.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+
+namespace LT.DigitalOffice.ProjectService.Validation
+{
+    public class AddUsersToProjectValidator : AbstractValidator<AddUsersToProjectRequest>
```
Comment (KostinP) [inline src/ProjectService.Validation/AddUsersToProjectValidator.cs:6]: Tests?

Comment (djino97) [reply src/ProjectService.Validation/AddUsersToProjectValidator.cs:6]: yes, i shall add in the short time

File: src/ProjectService.Validation/AddUsersToProjectValidator.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        public AddUsersToProjectValidator()
+        {
+            RuleFor(project => project.ProjectId)
+               .NotEmpty()
+               .WithMessage("Request must have a project Id");
+
+            When(project => project.Users != null, () =>
+            {
+                RuleForEach(project => project.Users).SetValidator(new ProjectUserRequestValidator());
+            });
```
Comment (KostinP) [inline src/ProjectService.Validation/AddUsersToProjectValidator.cs:17]: Not tested

Comment (djino97) [reply src/ProjectService.Validation/AddUsersToProjectValidator.cs]: fixed

**Review by KostinP — COMMENTED** [review]
Check what user exist?

**Comment by djino97** [comment]
> Check what user exist?

Fixed

**Review by MariyaSin — APPROVED** [review]

**Review by KostinP — APPROVED** [review]

**Pull request merged**